scripts/level/tr3/CHUNNEL.lua

```diff
@@ -4,7 +4,7 @@
 print("Level script loaded (CHUNNEL.lua)");
```

Comment:
nice! I think it is better to use 2 - 5 little PR instead one big - much more faster review and less merge conflicts, more faster master update.